### DISCUSSION TITLE
Unselecting after a search and replace

### DISCUSSION THREAD

**Discussion by englut** [helix-editor/helix#10081] [Q&A] [answered]
Hi there, I just figured out how to do a search and replace in a selection, but I can't seem to remove/stop the multiple selected mode. I often select multiple lines and replace specific words inside that selection. I really need an easy one quick keymap to exit this mode. Am I missing something from the documentation?

**Comment by joaander** [comment, marked as answer]
Press `,` to close the multiple selections and keep only the single primary selection. I learned this from the helix tutorial.

**Reply by englut** [reply]
Ahhh incomplete documentation, I see that note now. Must have glanced over it! I will quickly run through the tutorial. Thank you @joaander 🙏
